System.IO.Streams/StreamReader.cs

```diff
@@ -59,14 +60,19 @@ public bool EndOfStream
                 return _curBufLen == _curBufPos;
             }
         }
+        internal bool LeaveOpen
+        {
+            get { return !_closable; }
+        }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="StreamReader"/> class for the specified stream.
         /// </summary>
         /// <param name="stream">The stream to be read.</param>
+        /// <param name="leaveOpen">Leave stream open after dispose writer.</param>
```

Comment:
```suggestion
                /// <param name="leaveOpen"><see langword="true"/> to leave the stream open after the <see cref="StreamReader"/> object is disposed; otherwise, <see langword="false"/>.</param>
```